A/avrdude/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "avrdude"
+version = v"7.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/avrdudes/avrdude.git", "2e0be1e1aedfc10c904712bddda37e36356e7a66")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/avrdude/
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release
+make
```

Comment:
```suggestion
make -j${nproc}
```

Reply:
The build of this is so fast, it hardly seemed worth it - it took just 19s on CI and 1m total. The vast majority of that seems to be the download.